content/docs/deployment/csmoperator/modules/replication.md

```diff
@@ -56,3 +69,11 @@ To configure Replication perform the following steps:
 6. On the target cluster, configure the [prerequisites](../../../csmoperator/drivers/#pre-requisites-for-installation-of-the-csi-drivers) for deploying the driver via Dell CSM Operator.
 
 7. Install the CSI driver for your chosen storage platform on the source cluster according to the instructions for [installing the drivers using CSM Operator](../../../csmoperator/drivers/#installing-csi-driver-via-operator).
+
+8. _(Optional)_ If **CSM Replication** is deployed using two clusters in an environment where the DNS is not configured, it is necessary to update the dell-replication-controller-manager Kubernetes deployment to map the API endpoint FQDN to an IP address by adding the `hostAliases` field and associated FQDN:IP mappings.
```

Comment:
_CSM Replication_ does not need to be highlighted in this page since this page is about replication.